docker/images/e2e/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM marsha:dev as marsha
 
-FROM mcr.microsoft.com/playwright:jammy
+FROM mcr.microsoft.com/playwright:v1.33.0
```

Comment:
Same here, should not specify `-jammy` ? And this is not matching the version we are using in the back application